process/economics/economicsData.go

```diff
@@ -289,6 +305,13 @@ func (ed *economicsData) ComputeTxFee(tx process.TransactionWithFeeHandler) *big
 	return ed.ComputeMoveBalanceFee(tx)
 }
 
+// SplitTxGasInCategories returns the gas split per categories
+func (ed *economicsData) SplitTxGasInCategories(tx process.TransactionWithFeeHandler) (gasLimitMove, gasLimitProcess uint64) {
+	gasLimitMove = ed.ComputeGasLimit(tx)
+	gasLimitProcess = tx.GetGasLimit() - gasLimitMove
```

Comment:
could this be negative? E.g. for a move balance transaction that has some data, and the gasLimit is set to just 50000

Reply:
normally the tx should not be validated on the interceptor if it did not have enough gas to cover moveBalance + data.
so I would say no.
But can add a condition here as well.